### DISCUSSION TITLE
Is there an option to export the entire vectorstoreinfo database of the site that was 'learned' so that other users can just import the vectorized webpage?

### DISCUSSION THREAD

**Discussion by pleabargain** [langflow-ai/langflow#485] [Q&A]
Is there an option to export the entire vector database of the site that was 'learned' so that other users can just export/ import the vectorized webpage? 

Here's a screenshot for reference:
![image](https://github.com/logspace-ai/langflow/assets/640846/a6d5cb74-9871-4a26-9198-fa4ec4426c24)

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
